netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -467,6 +469,25 @@ private void gracefulClose(ChannelHandlerContext ctx, GracefulCloseCommand msg,
     close(ctx, promise);
   }
 
+  private void forcefulClose(final ChannelHandlerContext ctx, final ForcefulCloseCommand msg,
+      ChannelPromise promise) throws Exception {
+    lifecycleManager.notifyShutdown(
+        Status.UNAVAILABLE.withDescription("Channel requested transport to shut down"));
```

Comment:
Either `UNAVAILABLE` or `CANCELLED` makes sense. But yeah, I think we should be consistent with this Status and how we kill the streams below.